src/tilesheets.rs

```diff
@@ -329,12 +327,18 @@ impl TilesheetManager {
                     let name = format!("Tilesheet {} {} {}.png", self.name, tilesheet.size, z);
                     let path = BASE_PATH.join(name);
                     layer.save(&path).unwrap();
-                    Command::new("optipng").arg(path).spawn().unwrap()
+                    spawn(|| {
+                        oxipng::optimize(
+                            &oxipng::InFile::Path(path),
+                            &oxipng::OutFile::Path(None),
+                            &oxipng::Options::max_compression(),
```

Comment:
```suggestion
                            &oxipng::Options::max_compression(),
                            &oxipng::Deflaters::Zopfli(),
```

The [Zopfli](https://docs.rs/oxipng/2.2.2/oxipng/enum.Deflaters.html#variant.Zopfli) deflater is slightly more resource-intensive but results in smaller images that are comparable to OptiPNG.